packages/base/src/experiment-manager.ts

```diff
@@ -1,18 +1,19 @@
 import { Trace } from 'tsp-typescript-client/lib/models/trace';
-import { TspClient } from 'tsp-typescript-client/lib/protocol/tsp-client';
 import { Query } from 'tsp-typescript-client/lib/models/query/query';
 import { OutputDescriptor } from 'tsp-typescript-client/lib/models/output-descriptor';
 import { Experiment } from 'tsp-typescript-client/lib/models/experiment';
 import { TraceManager } from './trace-manager';
 import { TspClientResponse } from 'tsp-typescript-client/lib/protocol/tsp-client-response';
 import { signalManager, Signals } from './signals/signal-manager';
+import { TspFrontendClient } from './tsp-frontend-client';
+import { TspClient } from 'tsp-typescript-client';
```

Comment:
Minor; I would leave this import line unchanged at its original position.
```suggestion
import { TspClient } from 'tsp-typescript-client/lib/protocol/tsp-client';
```